File: update_env.py
import os

# Prompt for actual database credentials
mongo_uri = input("Enter your MongoDB URI (e.g., mongodb+srv://user:pass@cluster.mongodb.net/dbname): ").strip()
jwt_secret = input("Enter your JWT Secret: ").strip()
port = input("Enter PORT (default: 5000): ").strip() or "5000"
node_env = input("Enter NODE_ENV (default: development): ").strip() or "development"

if not mongo_uri:
    print("Error: MongoDB URI is required!")
    exit(1)

content = f"""PORT={port}
MONGO_URI={mongo_uri}
JWT_SECRET={jwt_secret}
NODE_ENV={node_env}"""

with open("d:/AI-Driven-CourseRecommendation System/server/.env", "w") as f:
    f.write(content)

print("✓ .env file created successfully!")

File: update_env.js
const fs = require('fs');
const content = `PORT=5000
MONGO_URI=mongodb://localhost:27017/ai_course_recommendation
JWT_SECRET=your_super_secret_jwt_key_change_in_production
NODE_ENV=development`;
fs.writeFileSync('d:/AI-Driven-CourseRecommendation System/server/.env', content);
console.log('Done');
